src/JBERC20.sol

```diff
@@ -35,18 +58,7 @@ contract JBERC20 is ERC20Votes, ERC20Permit, Ownable, IJBToken {
     // -------------------------- constructor ---------------------------- //
     //*********************************************************************//
 
-    /// @param name The name of the token.
-    /// @param symbol The symbol that the token should be represented by.
-    /// @param owner The owner of the token.
-    constructor(
-        string memory name,
-        string memory symbol,
-        address owner
-    )
-        ERC20(name, symbol)
-        ERC20Permit(name)
-        Ownable(owner)
-    {}
+    constructor() Ownable(address(this)) ERC20("", "") ERC20Permit("JBToken") {}
```

Comment:
Shouldn't we do something like 
```suggestion
    constructor() Ownable(address(this)) ERC20("", "") ERC20Permit("JBToken") {
        // Disable initialization on singleton instance.
        projectId = type(uint256).max;
    }
```

Reply:
srry, projectId is removed!!

Reply:
Perhaps if we use DrG suggestion about using the name and/or symbol as a stop for re-initialization we can do
```suggestion
    constructor() Ownable(address(this)) ERC20("invalid", "invalid") ERC20Permit("JBToken") {}
```

Reply:
i hadn't pushed up my local commits... @xBA5ED review the latest and lets see if this still makes sense. we use name, but just check for empty string.